aws-lambda-drt-removal/.eslintignore

```diff
@@ -0,0 +1,5 @@
+.circleci
```

Comment:
I assume you don't need `.circleci` and `terraform` seeing as you aren't including them in the repo?

Reply:
No, not required. I've now removed these.